repo/packages/A/arangodb/1/uninstall.json

```diff
@@ -0,0 +1,10 @@
+{
+  "zookeeper": {
+    "delete": [
+      {
+        "path": "/arangodb",
```

Comment:
I *think* we could do something like

```json
"path": "/arangodb/{{arangodb.id}}"
```

here to support removing just a specific instance of the framework but we would need to test it